apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -607,28 +609,49 @@ public Response getSymbols(   @Parameter(description = "Cryptocurrency identifie
                                 @Context HttpServletRequest req) throws NotFoundException {
 
         log.debug("getSymbols:  fsym: {}", symbol );
+        TimeZone tz = Calendar.getInstance().getTimeZone();                
         SymbolsOutputDTO symbolsOutputDTO = new SymbolsOutputDTO();
         symbolsOutputDTO.name = symbol;
         symbolsOutputDTO.exchange_traded = "Apollo Currency";
         symbolsOutputDTO.exchange_listed = "Apollo Currency";
-        symbolsOutputDTO.timezone = "America/New_York";
+        symbolsOutputDTO.timezone = tz.getID(); 
```

Comment:
exchange_traded&listed = Apollo DEX, because this information  describe an exchange on which trading for this asset was launched.

Reply:
done